docs/pages/management/admin/labels.mdx

```diff
@@ -74,7 +76,7 @@ as a resource:
    $ export INVITE_TOKEN=<token>
    ```
 
-## Step 2/3. Apply a static label
+## Step 2/4. Apply a static label
```

Comment:
I don't think these should be written as ordered steps. You don't need to apply static labels before you can apply dynamic labels, and you don't need to apply dynamic labels before you can apply ephemeral labels.

IMO it would make a lot more sense just to have a section for each of these topics rather than number them as if they are part of a specific sequence.